health/service/src/main/java/com/cerner/beadledom/health/HealthModule.java

```diff
@@ -116,8 +116,6 @@ protected void configure() {
     Multibinder<Module> jacksonModuleBinder = Multibinder.newSetBinder(binder(), Module.class);
     jacksonModuleBinder.addBinding().to(Jdk8Module.class);
     jacksonModuleBinder.addBinding().to(JavaTimeModule.class);
-
-    install(MultibindingsScanner.asModule());
```

Comment:
This was incorporated into default Guice as of 4.4. https://google.github.io/guice/api-docs/4.2/javadoc/com/google/inject/multibindings/MultibindingsScanner.html